pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -288,6 +297,66 @@ func (cloud *CloudProvider) insertRegionalDisk(ctx context.Context, volKey *meta
 	return nil
 }
 
+func (cloud *CloudProvider) insertRegionalAlphaDisk(ctx context.Context, volKey *meta.Key, diskType string, capBytes int64, capacityRange *csi.CapacityRange, replicaZones []string, snapshotID, diskEncryptionKmsKey string, multiWriter bool) error {
+	diskToCreateAlpha := &computealpha.Disk{
+		Name:        volKey.Name,
+		SizeGb:      common.BytesToGb(capBytes),
+		Description: "Regional disk created by GCE-PD CSI Driver",
+		Type:        cloud.GetDiskTypeURI(volKey, diskType),
+		MultiWriter: multiWriter,
+	}
+	if snapshotID != "" {
+		diskToCreateAlpha.SourceSnapshot = snapshotID
+	}
+	if len(replicaZones) != 0 {
+		diskToCreateAlpha.ReplicaZones = replicaZones
+	}
+	if diskEncryptionKmsKey != "" {
+		diskToCreateAlpha.DiskEncryptionKey = &computealpha.CustomerEncryptionKey{
+			KmsKeyName: diskEncryptionKmsKey,
+		}
+	}
+
+	insertOp, err := cloud.alphaService.RegionDisks.Insert(cloud.project, volKey.Region, diskToCreateAlpha).Context(ctx).Do()
+	if err != nil {
+		if IsGCEError(err, "alreadyExists") {
+			disk, err := cloud.GetDisk(ctx, volKey)
+			if err != nil {
+				return err
+			}
+			err = cloud.ValidateExistingDisk(ctx, disk, diskType,
```

Comment:
validation might also need to be changed to check if `multiWriter` field is equal